pkg/trace/testutil/otlp.go

```diff
@@ -87,6 +98,17 @@ func setOTLPSpan(span ptrace.Span, s *OTLPSpan) {
 		insertAttributes(ev.Attributes(), e.Attributes)
 		ev.SetDroppedAttributesCount(e.Dropped)
 	}
+	links := span.Links()
+	for _, l := range s.Links {
+		li := links.AppendEmpty()
+		bytes, _ := hex.DecodeString(l.TraceID)
```

Comment:
I wouldn't ignore the error. I'd rather `panic`. It's a test, a panic won't be a tragedy. However having an error here for whatever odd reason can cause a really long debugging session otherwise :) Same for the below.

Reply:
Done.